lib/components/admin/mailables-window.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react'
+import { Badge } from 'react-bootstrap'
+import { connect } from 'react-redux'
+
+import * as callTakerActions from '../../actions/call-taker'
+import DraggableWindow from './draggable-window'
+import Icon from '../narrative/icon'
+import {MailablesList, WindowHeader} from './styled'
+import {createLetter, MAILABLE_FIELDS} from '../../util/mailables'
```

Comment:
lol, we really need a lint/prettier rule about bracket spacing. I don't care what spacing we use, but it sure would be nice to have things be consistent.